webpack.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  entry: './js/entries/all.js',
+  output: {
+      path: './_build/assets/js/',
+      filename: 'foundation.js'
+  },
+  externals: {
+    jquery: 'jQuery'
+  },
+  module: {
+    loaders: [
+    ],
+    rules: [
+      // JS LOADER
+      // Reference: https://github.com/babel/babel-loader
+      // Transpile .js files using babel-loader
+      // Compiles ES6 and ES7 into ES5 code
+      {
```

Comment:
Moving to Webpack opens up the possibility to modularize Foundation more in the future. Like, move independently-useful plugins to separate repos, where they can have their own documentation and tests and so on. If we ever do that, be sure to add `exclude: /node_modules/` to this Babel rule so those dependencies aren't transpiled. That's standard practice when using babel-loader.